qiskit/ignis/experiments/calibration/generators/circuit_based.py

```diff
@@ -0,0 +1,31 @@
+from typing import Iterable, Optional, Union, List
+
+from qiskit import QuantumCircuit
+
+from qiskit.ignis.experiments.calibration.cal_base_generator import Base1QCalibrationGenerator
+
+
+class CircuitBasedGenerator(Base1QCalibrationGenerator):
```

Comment:
Why is the `CircuitBasedGenerator` derived from 1Q? We need another `CircuitBasedGenerator` for 2Q?

Reply:
I think we will want to refactor/rename this.